src/LiveComponent/assets/package.json

```diff
@@ -27,7 +27,7 @@
         }
     },
     "dependencies": {
-        "idiomorph": "^0.3.0"
+        "idiomorph": "https://github.com/bigskysoftware/idiomorph.git"
```

Comment:
Sure about this ?

Reply:
Yup, but good question. Copied from Turbo - it allows us to get the absolute latest version. Will change after they tag next. We embed this library (for now), so this isn't exposed to the user.